_i18n/en/general/privacy.md

```diff
@@ -1,34 +1,53 @@
 AntennaPod is developed and maintained by individual volunteers and not represented by any legal entity. The community doesn't need your data, so the app and website are designed to be fully GDPR compliant. Read on to learn more.
 
-## What kind of data we store
-The AntennaPod app only stores data strictly necessary to for the app to function, as provided by you, the user. This includes for example the list of podcasts you follow, the app settings, and authentication data provided for specific podcasts or services (such as Gpodder). The app does not include any advertisement libraries or any 3rd party tracking (analytics) code, such as Google Analytics.
+## What data the AntennaPod app might collect, store and process
+The app only stores data strictly necessary to for the app to function. This includes for example the list of podcasts you follow, the app settings, and authentication data provided for specific podcasts or services (such as synchronisation services). The app does not include any advertisement libraries or any 3rd party tracking (analytics) code, such as Google Analytics.
 
-The AntennaPod website, hosted via GitHub Pages, does not store any cookies on your machine, and no 3rd party tracking (analytics) or other services are used.
+**All data provided and created when using the AntennaPod app is stored locally on your device. AntennaPod does not upload your data anywhere, except where strictly necessary for the functioning of the app.** Please see the second next section for the cases in which your data may be provided to third parties.
```

Comment:
Doesn't this basically say the same thing twice?

> only stores data strictly necessary to for the app to function

> except where strictly necessary for the functioning of the app

Reply:
Kind of, though they apply to different things: one is about recording data, the other about uploading data.